Makefile

```diff
@@ -126,7 +126,7 @@ install_plugins: .pulumi/bin/pulumi
 	.pulumi/bin/pulumi plugin install resource random 4.8.2
 	.pulumi/bin/pulumi plugin install resource github 5.14.0
 	.pulumi/bin/pulumi plugin install resource std 1.6.2
-	.pulumi/bin/pulumi plugin install converter terraform 1.0.16
```

Comment:
Looks like this change might be causing the diff in the descriptions

Reply:
It should, since this is what we use to generate code examples.